core/src/main/java/bisq/core/dao/node/parser/TxParser.java

```diff
@@ -330,8 +330,10 @@ static boolean isTxInvalid(TempTx tempTx, boolean bsqOutputFound, boolean burntB
             return true;
         }
 
-        if (!bsqOutputFound) {
-            log.warn("Invalid Tx: No BSQ output found. tx=" + tempTx);
+        if ((tempTx.getTxType() == TxType.COMPENSATION_REQUEST ||
+                tempTx.getTxType() == TxType.REIMBURSEMENT_REQUEST)
+                && !bsqOutputFound) {
```

Comment:
I think this is better than setting any tx with `!bsqOutputFound` to invalid.

Reply:
Yes it was needed as otherwise we would enfore a mandatory BSQ change for all BSQ txs. For the comp request we need it to make parsing easier with the BTC candidate output but for others not.